Packs/RecordedFuture/Integrations/RecordedFuture/README.md

```diff
@@ -708,6 +708,116 @@ Get details on alerts configured and generated by Recorded Future by alert rule
 >| Global Trends, Trending Targets - Spike: Enel SPA, Knoxville and Alabama |
 
 
+### recordedfuture-alert-set-status
```

Comment:
Please add a note fin the README file for the `fetch-incidents` command you added.